README.md

```diff
@@ -10,19 +10,43 @@ A realization of the popular trading card game - yugioh on web using React + Nod
 
 That's right, no ygocore and its Lua card scripts but a completely new Yugioh simulator written in Javascript. You can also access its desktop version just by wrapping the source code with some simple tweaks using Electron (just git checkout the `electron` branch lol)!
 
-## Installation & How to run
-```
-git clone https://github.com/rickypeng99/yugioh_web_server.git
-cd /yugioh_web_server
-node index.js
-
-git clone https://github.com/rickypeng99/yugioh_web.git
-```
-Open two terminal windows up and run the following in each of them
-```
-npm start
-```
-GLHF!
+## Installation for Development
+1. `git clone https://github.com/rickypeng99/yugioh_web_server.git`
+2. Change directories to `./yugioh_web_server`
+3. Command: `npm install`
+4. `git clone https://github.com/BBeretta/yugioh_web.git`
+4. Change directories to `./yugioh_web`
+5. Command: `npm install`
+
+## Running Server
+1. Open two terminal windows up and run the following in each of them
+2. Change directories to `./yugioh_web_server`
+3. `node index.js`
```

Comment:
Thanks for the PR. Are you sure we need to run the server twice? I have an impression that the server is centralized

Reply:
Hey, sorry! 
My mistake! Copy and Paste fails :P

Reply:
Np, great work!